### PR TITLE
Rke2june2023update

### PR DESCRIPTION
**Pull request by briandowns**

### CODE REVIEW

**Review by galal-hussein — DISMISSED** [review]

File: channels-rke2.yaml [hunk 1]
```diff
@@ -1152,4 +1152,20 @@ releases:
         repo: rancher-rke2-charts
         version: 2.11.100-build2023051508
     featureVersions: *featureVersions-v1
+  - version: v1.24.15+rke2r1
+    minChannelServerVersion: v2.7.5-alpha1
```
Comment (rancher-max) [inline channels-rke2.yaml]: This should be `v2.6.7-alpha1`

File: channels-rke2.yaml [hunk 2]
```diff
@@ -1152,4 +1152,20 @@ releases:
...
+    maxChannelServerVersion: v2.7.99
```
Comment (rancher-max) [inline channels-rke2.yaml]: This should be `v2.6.99`

**Review by rancher-max — CHANGES_REQUESTED** [review]
After addressing a couple minor changes, this looks fine from the rke2 perspective. However, I'm not sure why a bunch of other files got added in? Might need a review from @kinarashah or @snasovich to see if there was a change to the 2.6 branch and if these other files are expected now?

**Comment by kinarashah** [comment]
@rancher-max We possibly need a rebase. ACI changes are already present in the `dev-v2.6-2023-06-patches` branch.

**Comment by briandowns** [comment]
Rebased.

**Review by rancher-max — APPROVED** [review]

**Review by kinarashah — APPROVED** [review]

**Pull request merged**